### PR TITLE
Updated contribute/containers _index.md

### PR DESCRIPTION
**Pull request by Rutam21**
#### Summary

The links were directly written onto the page which was not looking nice. Proper formatting has been provided for the links.

#### Ticket Link

NA

### CODE REVIEW

**Comment by Rutam21** [comment]
@cwarnermm Please review and add the necessary labels for Docathon.

**Review by justinegeffen — COMMENTED** [review]
Thank you for this PR and for noting the inconsistencies in how URLs are displayed.

I have reviewed the live version and noted that toward the bottom of the page, similar entries are presented as: "Please refer to the [GitHub repo](URL) for more information." (where "URL" is replaced by the URL.

I recommend using this format for consistency and clarity. It should fit in all the examples although if a link is not to a GitHub repo I'd recommend changing the wording to match the target. 

Let me know if you have any questions!

File: site/content/contribute/containers/_index.md [hunk 1]
```diff
@@ -10,50 +10,50 @@ This page lists all the Docker repositories currently in use.
...
 
 - [mattermost/podman](https://hub.docker.com/repository/docker/mattermost/podman) - For internal use. Contains Podman to build/tag/push container images.
 
 - [mattermost/chewbacca](https://hub.docker.com/repository/docker/mattermost/chewbacca-bot) - For internal use. A GitHub Bot for administrative tasks. Please refer to the [GitHub repo](https://github.com/mattermost/chewbacca) for more information.
 
 - [mattermost/matterwick](https://hub.docker.com/repository/docker/mattermost/matterwick) - For internal use. A GitHub Bot to spin test servers for pull requests. Please refer to the [GitHub repo](https://github.com/mattermost/matterwick) for more information.
 
-- [mattermost/webrtc](https://hub.docker.com/repository/docker/mattermost/webrtc) - DEPRECATED. Preview docker image of Mattermost WebRTC. Docs: http://bit.ly/2fVbWjU
+- [mattermost/webrtc](https://hub.docker.com/repository/docker/mattermost/webrtc) - DEPRECATED. Preview docker image of Mattermost WebRTC. Docs: [Here](http://bit.ly/2fVbWjU)
```
[the remark this answers is not in the document]
Comment (Rutam21) [reply site/content/contribute/containers/_index.md:59]: Yeah, sure. reverting those changes.

**Comment by Rutam21** [comment]
@justinegeffen The suggested changes are done. Please review.

File: site/content/contribute/containers/_index.md [hunk 2]
```diff
@@ -10,50 +10,50 @@ This page lists all the Docker repositories currently in use.
...
 
 - [mattermost/mattermost-push-proxy](https://hub.docker.com/r/mattermost/mattermost-push-proxy) - Mattermost Push Proxy. [Documentation](https://developers.mattermost.com/contribute/mobile/push-notifications/service/). [GitHub](https://github.com/mattermost/mattermost-push-proxy).
 
 - [mattermost/mattermost-loadtest](https://hub.docker.com/r/mattermost/mattermost-loadtest) - Image for the Load Test application. Tools for profiling Mattermost under heavy load. [GitHub](https://github.com/mattermost/mattermost-load-test).
 
-- [mattermost/mattermost-operator](https://hub.docker.com/r/mattermost/mattermost-operator) - Official image for Mattermost Operator for Kubernetes. For more information see https://github.com/mattermost/mattermost-operator.
+- [mattermost/mattermost-operator](https://hub.docker.com/r/mattermost/mattermost-operator) - Official image for Mattermost Operator for Kubernetes. For more information check the [GitHub repo](https://github.com/mattermost/mattermost-operator).
```
Comment (justinegeffen) [inline site/content/contribute/containers/_index.md:21]:
```suggestion
- [mattermost/mattermost-operator](https://hub.docker.com/r/mattermost/mattermost-operator) - Official image for Mattermost Operator for Kubernetes. For more information please refer to the [GitHub repo](https://github.com/mattermost/mattermost-operator).
```

File: site/content/contribute/containers/_index.md [hunk 3]
```diff
@@ -10,50 +10,50 @@ This page lists all the Docker repositories currently in use.
...
 
-- [mattermost/mattermost-cloud](https://hub.docker.com/r/mattermost/mattermost-cloud) - Mattermost Private Cloud is a SaaS offering meant to smooth and accelerate the customer journey from trial to full adoption. For more information see https://github.com/mattermost/mattermost-cloud.
+- [mattermost/mattermost-cloud](https://hub.docker.com/r/mattermost/mattermost-cloud) - Mattermost Private Cloud is a SaaS offering meant to smooth and accelerate the customer journey from trial to full adoption. For more information check the [GitHub repo](https://github.com/mattermost/mattermost-cloud).
```
Comment (justinegeffen) [inline site/content/contribute/containers/_index.md:23]:
```suggestion
- [mattermost/mattermost-cloud](https://hub.docker.com/r/mattermost/mattermost-cloud) - Mattermost Private Cloud is a SaaS offering meant to smooth and accelerate the customer journey from trial to full adoption. For more information please refer to the [GitHub repo](https://github.com/mattermost/mattermost-cloud).
```

**Review by justinegeffen — APPROVED** [review]
Thank you for this PR! :)

**Review by cwarnermm — APPROVED** [review]
Thank you for your docs contribution!

**Pull request merged**